kubernetes/monitoring/prometheus-values.yaml

```
@@ -0,0 +1,176 @@
+      repeat_interval: 12h
+      receiver: 'slack'
+      routes:
+      - match:
+          alertname: Watchdog
+        receiver: 'null'
+      - match:
+          severity: critical
+        receiver: 'slack'
+        continue: true
+    receivers:
+    - name: 'null'
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)